platform/podjs/src/manifest-parser/src/manifest.ts

```diff
@@ -67,22 +57,24 @@ const featureDecoder = Decode.type({
     translations: Decode.UnknownRecord,
 });
 
-export async function readManifest(packageManifest: Record<string, unknown>): Promise<Manifest> {
-    // There is no 'version' property in real feature manifests at this time, but to make this code
-    // happy, we're simply adding one if it's missing.
-    const manifestWithVersion = { ...packageManifest };
-    if (!("version" in manifestWithVersion)) manifestWithVersion.version = "1.0.0";
+export async function readManifest(
+    packageManifest: Record<string, unknown>
+): Promise<Manifest> {
+    const manifest = { ...packageManifest };
 
-    const rawMain = expect(manifestWithVersion, "Failed to parse main manifest", mainDecoder);
+    const rawMain = expect(
```

Comment:
Have you considered merging these two decoders into one, and these two `expect` into one? It does not make sense anymore, as long as the `mainManifest` is simply a string.